static/src/javascripts/projects/common/modules/experiments/tests/mt-sticky-nav.js

```diff
@@ -0,0 +1,158 @@
+define([
+    'common/utils/$',
+    'common/utils/_',
+    'common/utils/config',
+    'common/utils/detect',
+    'common/utils/mediator'
+], function (
+    $,
+    _,
+    config,
+    detect,
+    mediator
+) {
+    return function () {
+        this.id = 'MtStickyNav';
+        this.start = '2015-03-19';
+        this.expiry = '2015-05-19';
+        this.author = 'Zofia Korcz';
+        this.description = 'Top navigation and top ad slot are sticky';
+        this.audience = 0.01;
+        this.audienceOffset = 0.3;
```

Comment:
I think we have some over tests current running in this offset segment. Are you happy for it to clash?